src/GitPrompt.ps1

```diff
@@ -132,7 +132,7 @@ if (Get-Module NuGet) {
 
 function Write-Prompt($Object, $ForegroundColor = $null, $BackgroundColor = $null) {
     $s = $global:GitPromptSettings
-    if ($s -and !$ForegroundColor) {
+    if ($s -and $ForegroundColor -eq $null) {
```

Comment:
Good catch!  To eliminate a PSScriptAnalyzer warning in VSCode, would you mind changing that ever so slightly to:
```posh
if ($s -and ($null -eq $ForegroundColor)) { 
```